src/Quasar/Advanced/Types.purs

```diff
@@ -464,6 +464,37 @@ instance encodeJsonProvider ∷ EncodeJson Provider where
     ~> "client_id" := Newtype.unwrap obj.clientId
     ~> "openid_configuration" := OpenIDConfiguration obj.openIDConfiguration
     ~> jsonEmptyObject
+    
+data LicenseStatus = LicenseValid | LicenseExpired
+
+decodeLicenseStatus ∷ Json → Either String LicenseStatus
+decodeLicenseStatus =
+  decodeJson >=>
```

Comment:
instead of having a function which doesn't start with decodeJson is there a way to combine the two nicely?